internal/_os_test.ts

```diff
@@ -0,0 +1,65 @@
+// Copyright 2018-2025 the Deno authors. MIT license.
+import { assertEquals } from "@std/assert/equals";
+import { checkWindows } from "./_os.ts";
+import { disposableStack, stubProperty } from "./_testing.ts";
+
+// deno-lint-ignore no-explicit-any
+const global = globalThis as any;
+
+Deno.test("checkWindows() returns `true` on Windows on various runtimes", async (t) => {
+  await t.step("Deno", () => {
+    using stack = disposableStack();
+    stack.use(stubProperty(global, "Deno", { build: { os: "windows" } }));
+    stack.use(stubProperty(global, "navigator", undefined));
+    stack.use(stubProperty(global, "process", undefined));
```

Comment:
Is this better than 3 using statements? (Do we need `DisposableStack` polyfill?)

```suggestion
    using _denoStub = stubProperty(global, "Deno", { build: { os: "windows" } });
    using _navigatorStub = stubProperty(global, "navigator", undefined);
    using _processStub = stubProperty(global, "process", undefined);
```

Reply:
Not much in it except for importing the polyfill vs creating 3 unused variables instead of one. `DisposableStack` is supported in latest Deno, only partial-polyfilled for testing due to lack of support in Deno 1.x, Node, etc. (the other usage is in `mockLittleEndian` in `random/_test_utils.ts`, which uses `defer` instead of `use`).

Reply:
Ok. Makes sense